After poolstl was compiled, the `poolstl.lib` file was not found in the package and installed directories. And the following error occurred when testing usage.
```
Severity	Code	Description	Project	File	Line	Suppression State	Details
Error		CMake Error at F:/poolstl/installed/x64-windows/share/poolstl/poolSTLTargets.cmake:60 (set_target_properties):
  The link interface of target "poolSTL::poolSTL" contains:

    Threads::Threads

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.		F:/poolstl/installed/x64-windows/share/poolstl/poolSTLTargets.cmake	60		

```

--- a/ports/poolstl/fix-find-dependency.patch
+++ b/ports/poolstl/fix-find-dependency.patch
@@ -1,0 +1,14 @@
+diff --git a/cmake/config.cmake.in b/cmake/config.cmake.in
+--- a/cmake/config.cmake.in
++++ b/cmake/config.cmake.in
+@@ -1,7 +1,10 @@
+ 
+ @PACKAGE_INIT@
+ 
++include(CMakeFindDependencyMacro)
++find_dependency(Threads)
++
+ include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+ 
+ set_and_check(@PROJECT_NAME@_INCLUDE_DIR "@PACKAGE_CMAKE_INSTALL_INCLUDEDIR@")
+ check_required_components("@PROJECT_NAME@")

--- a/ports/poolstl/portfile.cmake
+++ b/ports/poolstl/portfile.cmake
@@ -4,6 +4,7 @@ vcpkg_from_github(
     REF "v${VERSION}"
     SHA512 a2d29056b29f32f034560f88e05f7257ff1f43b46579b940da3e340c97cf8bfbb7d886f5101044d5e22931af37bbcf72f956a0157e135cdf10c7a987e56ba081
     HEAD_REF main
+    PATCHES fix-find-dependency.patch
 )
 
 vcpkg_cmake_configure(

--- a/ports/poolstl/portfile.cmake
+++ b/ports/poolstl/portfile.cmake
@@ -1,10 +1,13 @@
+set(VCPKG_BUILD_TYPE release) # header-only
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO alugowski/poolSTL
     REF "v${VERSION}"
     SHA512 a2d29056b29f32f034560f88e05f7257ff1f43b46579b940da3e340c97cf8bfbb7d886f5101044d5e22931af37bbcf72f956a0157e135cdf10c7a987e56ba081
     HEAD_REF main
-    PATCHES fix-find-dependency.patch
+    PATCHES
+        fix-find-dependency.patch
 )
 
 vcpkg_cmake_configure(
@@ -13,7 +16,5 @@ vcpkg_cmake_configure(
 
 vcpkg_cmake_install()
 vcpkg_cmake_config_fixup(CONFIG_PATH share/cmake/poolSTL)
-
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
 
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE-BSD.txt" "${SOURCE_PATH}/LICENSE-Boost.txt" "${SOURCE_PATH}/LICENSE-MIT.txt")

--- a/versions/p-/poolstl.json
+++ b/versions/p-/poolstl.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "36c335e265bfdd1065fa1fb81961ac8aa6b70213",
+      "git-tree": "a60a15e3420a2693f2e7080ae41808411abcbf36",
       "version": "0.3.5",
       "port-version": 0
     }

